src/main/java/uk/ac/ebi/ampt2d/accession/ApplicationConstants.java

```diff
@@ -0,0 +1,27 @@
+/*
+ *
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.ampt2d.accession;
+
+public class ApplicationConstants {
+
+    public static final String APPLICATION_INSTANCE_ID = "accessioning.instance.id";
```

Comment:
I commented this in other project, I know that @jorizci sometimes likes to have the constants grouped in a class. I see them belonging to other classes like EvaAccessionApplication. Any other opinion? I din't dislike it so much, so is OK if this class remains.

Reply:
Actually, they are being used in VariantAccessioningConfiguration, so maybe that is the place